src/kservice.c

```diff
@@ -93,13 +93,13 @@ rt_weak void rt_hw_cpu_shutdown(void)
 
 rt_weak rt_err_t rt_hw_backtrace_frame_get(rt_thread_t thread, struct rt_hw_backtrace_frame *frame)
 {
-    LOG_W("%s: not implemented");
+    LOG_W("not implemented");
```

Comment:
处理成这样吧，否则用户并不知道哪个函数未实现

```suggestion
    LOG_W("%s is not implemented", __func__);
```